src/node/jsonRpc/tests/Utils/BlockFinder.test.js

```diff
@@ -0,0 +1,97 @@
+import { expect, assert } from 'chai';
+
+import BlockFinder from '../../Utils/BlockFinder';
+
+describe('BlockFinderTest', () => {
+    const Blockchain = {
```

Comment:
This object can be a mock of the actual Blockchain class also, but this way is cleaner